ogr/abstract.py

```diff
@@ -315,6 +315,17 @@ class PRStatus(IntEnum):
     all = 4
 
 
+class CommitStatus(Enum):
+    pending = 1
+    success = 2
+    failure = 3
+    error = 4
+    canceled = 5
+    running = 6
+    mergeable = 7
+    notmergeable = 8
```

Comment:
I'm not sure merging those two use-cases together is a good idea. I can't think of any use where both `mergeable` and `success`/`failure`/.. makes sense.

I think others suggested using a new enum and get inspired by `CommitStatus`. Not to reuse `CommitStatus`.